ui/advanced-widgets.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Advanced Custom Widgets
+---
+
+# Custom Widgets - Advanced Topics
+
+[[toc]]
+
+This page expands of some of the topics initially introduced in [Building Pages](building-pages.html).
+
+## Advanced Components
+
+In the custom widget system there are many options for different components to include beyond the standard OH set.  Where each component comes from determines many of the options available for configuration and styling.
+
+### F7 components
+
+The MainUI is built primarily using the [Framework7](https://framework7.io/) library.  Because most of the OH specific components are built on top of these f7 components, [the documentation of the f7 components](https://framework7.io/vue/) is useful for users wishing to understand more about the OH components as well.  In addition, however, the f7 components themselves are available as options in the widget editor.
+
+As a general rule, the f7 components will have more configuration and style flexibility than the OH counterparts.  So, their use is recommended when there is something about the component that needs to be configured in a way different than what is set in the OH version.  Of course, the f7 components do not have the OH specific functions available, so while they can have values based on items using the epxression system, they cannot easily be used to trigger rules, update items or variables, etc.
```

Comment:
I personally would prefer to capitalize `Items` as this is an OH term.
This is not consistent across the docs, but most newer doc changes capitalize OH terms like `Items`, `Things`, ….

Again, put each sentence on its own line.